pubmed_crawler.py

```diff
@@ -155,8 +162,10 @@ def parse_header(header):
     doi = doi_cit.text.strip().lstrip("doi: ").rstrip(".") if doi_cit else ""
 
     # AUTHORS
-    authors = [a.find('a').text for a in header.find_all(
-        'span', attrs={'class': "authors-list-item"})]
+    authors = [
+        a.find('a').text
+        for a in header.find_all('span', attrs={'class': "authors-list-item "})
+    ]
```

Comment:
I think we should handle the case in which there _isn't_ a space as well, so something like this:

```python
authors = [a.find('a').text for a in header.find_all(
        'span', attrs={'class': "authors-list-item"})]
if not authors:
   authors = [a.find('a').text for a in header.find_all(
        'span', attrs={'class': "authors-list-item "})]
```